oktapam/data_source_ad_connection.go

```diff
@@ -2,9 +2,8 @@ package oktapam
 
```

Comment:
@sachinsaxena-okta I didn't want to touch the AD connection and gateway data sources because I wanted to leave you to refactor them in a separate PR.

Reply:
I will do this.